### DISCUSSION TITLE
How do i get it to use my username instead of my name?

### DISCUSSION THREAD

**Discussion by Sopor** [anuraghazra/github-readme-stats#3601] [Q&A] [answered]
I want it to use my username instead of my name. How do i do that?

![image](https://github.com/anuraghazra/github-readme-stats/assets/5789283/831fcc9a-cf41-4385-880a-96b6dda46412)

Instead of using `Anurag Hazra` i want it to use `anuraghazra`

Is that possible?

**Comment by qwerty541** [comment, marked as answer]
Hey, @Sopor! You con modify title by using `custom_title` query option. Here is an example:

```html
<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&custom_title=anuraghazra%27s%20GitHub%20Stats" />
```

<img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&custom_title=anuraghazra%27s%20GitHub%20Stats" />
